nntrainer/models/neuralnet.cpp

```diff
@@ -258,13 +258,13 @@ int NeuralNetwork::initialize() {
     }
   }
   setBatchSize(batch_size);
+  // Allocate and initialize weights
+  manager->initializeWeights();
```

Comment:
I wonder about a reason for the rearrangement.
Is it because inPlaceOptimize should have to happen after initializeWeights?

Reply:
This is because `readModels` should happen after `initializeWeights`, but before `train/inference`.
However, gradient is not requried in inference. Hence the rearrangement.

Reply:
Noted Thanks